extensions/grunt/src/main.ts

```diff
@@ -101,6 +116,20 @@ class FolderDetector {
 		return this.promise;
```

Comment:
All of `getTasks` needs to be wrapped in an `isEnabled` check. See the Gulp extension.